cob_moveit_config/raw3-1/config/ompl_planning.yaml

```diff
@@ -30,10 +30,9 @@ planner_configs:
     goal_bias: 0.05  # When close to goal select goal, with this probability? default: 0.05
   RRTConnectkConfigDefault:
     type: geometric::RRTConnect
-    range: 0.1  # Max motion added to tree. ==> maxDistance_ default: 0.0, if 0.0, set on setup()
+    range: 0.0  # Max motion added to tree. ==> maxDistance_ default: 0.0, if 0.0, set on setup()
   RRTstarkConfigDefault:
     type: geometric::RRTstar
-    optimization_objective: MaximizeMinClearanceObjective
```

Comment:
why did you change these values?
do the previous not result in a satisfying behavior?